### ISSUE TITLE
Cannot enable file-based logging

### ISSUE THREAD

**Issue by mtdeguzis** [arthurdejong/nss-pam-ldapd#34]
https://arthurdejong.org/nss-pam-ldapd/nslcd.conf.5
```
nslcd: /etc/nslcd.conf:8: unknown keyword: 'log'
```

/etc/nslcd.conf
```
# logging
log /var/log/nslcd.log debug
```

Version:
````
[mtdeguzis@HOST: ~]$ nslcd --version
nss-pam-ldapd 0.8.13
Written by Luke Howard and Arthur de Jong.

Copyright (C) 1997-2013 Luke Howard, Arthur de Jong and West Consulting
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

**Comment by quanah** [comment]
I would suggest updating to a current release.  The version you quote is at least 5 years old.  It's unlikely the man page from the current release you quoted is fully applicable.

```
Latest news

    2018-09-01: Release 0.9.10 of nss-pam-ldapd
    2018-02-18: Release 0.9.9 of nss-pam-ldapd
    2017-06-26: Release 0.9.8 of nss-pam-ldapd
    2014-03-16: Release 0.8.14 of nss-pam-ldapd
```

**Comment by mtdeguzis** [comment]
oh wow, RHEL strikes again! You can close if our version just doesn't have that option.

**Comment by arthurdejong** [comment]
I can confirm that the log option is only available in the 0.9 series.
